src/hooks/useDragAndDrop.js

```diff
@@ -58,6 +58,7 @@ export default function useDragAndDrop({dropZone, onDrop = () => {}, shouldAllow
             }
 
             event.preventDefault();
+            event.stopPropagation();
```

Comment:
We need it for the drag and drop to work on the tab as there is a `<NoDropZone>` component and a nested drag and drop on the chat page.

Reply:
More generically, when we have nested drag and drop zones, this makes it so that the parent drop zones don't interfere with ones further down in the tree, which I think will generally make the feature less prone to bugs